src/utils/SelectOption.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+
+export default class SelectOption {
```

Comment:
Why do we need it?
let's discuss

Reply:
I haven't used it yet. I would use common model for a few components and don't want to duplicate propTypes across them.
Where is a good place for that?

Reply:
I believe we can try to use `centralized PropTypes` react pattern in this case 

please have a look an example 
https://www.freecodecamp.org/news/react-pattern-centralized-proptypes-f981ff672f3b/

What do you think?

Reply:
Thank you for link! It is definitely what I need.